pkg/render/apiserver.go

```diff
@@ -568,6 +568,16 @@ func (c *apiServerComponent) apiServerContainer() corev1.Container {
 
 	isPrivileged := true
 
+	env := []corev1.EnvVar{
+		{Name: "DATASTORE_TYPE", Value: "kubernetes"},
+	}
+
+	if c.installation.Spec.Variant == operator.TigeraSecureEnterprise {
```

Comment:
The apiserver is an enterprise only component, so don't need this check.

Reply:
Also, we should prevent setting `MultiInterfaceMode` to non-nil for Calico OSS clusters in validation anyway

Reply:
Right, I think we do but I'll double check

Reply:
We don't actually, but I'll make a separate PR to handle that.